linera-base/src/crypto/secp256k1/evm.rs

```diff
@@ -0,0 +1,635 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
(I think we omit this for new files? Or is it because it copies some code from the other file?)

Reply:
Ah, this particular line you mean? I didn't check in details the error message.

Reply:
Yes; I guess if it's all new code, the copyright is only Zefchain.